test/v2/controllers/requestParsers/validators/CreateForeignPropertyPeriodSummaryValidatorSpec.scala

```diff
@@ -518,6 +576,66 @@ class CreateForeignPropertyPeriodSummaryValidatorSpec extends UnitSpec with Mock
           )) shouldBe List(ToDateFormatError)
 
       }
+
+      "the toDate is out of range" in {
+        val validator = setUpValidator()
+        validator.validate(
+          CreateForeignPropertyPeriodSummaryRawData(
+            validNino,
+            validBusinessId,
+            validTaxYear,
+            Json.parse(
+              """
+                |{
+                |   "fromDate":"2019-01-01",
```

Comment:
As the json appears to be largely the same in each of these tests, perhaps they could be fetched from a "private def" (stuffed down at the end of the test class) and parameterised with the fromDate & toDate?

Reply:
@molafisoye My comment above is also same